### PR TITLE
Add XSIAM part of Veeam App

### PR DESCRIPTION
**Pull request by nikita-konoplin**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Veeam/ReleaseNotes/1_0_1.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#### XSIAM Dashboards
+
+##### New: Veeam Data Platform Monitoring
+Added the **'Veeam Data Platform Monitoring'** XSIAM dashboard for Veeam App pack.
+##### New: Veeam Security Activities
+Added the **'Veeam Security Activities'** XSIAM dashboard for Veeam App pack.
```
Comment (ShirleyDenkberg) [inline Packs/Veeam/ReleaseNotes/1_0_1.md:6]:
```suggestion
Added the **'Veeam Data Platform Monitoring'** Cortex XSIAM dashboard for the Veeam App pack.
##### New: Veeam Security Activities
Added the **'Veeam Security Activities'** Cortex XSIAM dashboard for the Veeam App pack.
```

File: Packs/Veeam/README.md [hunk 2]
```diff
@@ -1,8 +1,14 @@
-The Veeam App for Palo Alto Networks XSOAR allows Veeam Data Platform Advanced and Premium customers to combine the automation and orchestration features of Cortex XSOAR with a simple and powerful [Veeam Data Platform](https://www.veeam.com/products/veeam-data-platform.html) that goes beyond backup, providing businesses with reliable data protection, seamless recovery, and streamlined data management. Using the data received from Veeam Backup & Replication and Veeam ONE REST APIs the Veeam App creates custom incidents related to malware detection and the health state of the backup infrastructure components. These incidents can be managed through built-in Veeam Incident Dashboard and resolved manually or automatically with built-in Veeam playbooks.
+# Overview
+
+Secure Backup is critical to your cyber resilience. [Veeam Data Platform](https://www.veeam.com/products/veeam-data-platform.html) provides comprehensive capabilities to extend the principles of Zero Trust to data backup and recovery including Proactive Threat Hunting, Immutability Everywhere, and Secure Access.
```
Comment (ShirleyDenkberg) [inline Packs/Veeam/README.md:3]:
```suggestion
Secure backup is critical to your cyber resilience. [Veeam Data Platform](https://www.veeam.com/products/veeam-data-platform.html) provides comprehensive capabilities to extend the principles of Zero Trust to data backup and recovery including Proactive Threat Hunting, Immutability Everywhere, and Secure Access.
```

File: Packs/Veeam/README.md [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
+
+<~XSOAR>
+Using the data received from Veeam Backup & Replication and Veeam ONE REST APIs, the App creates custom incidents related to malware detection and the health state of the backup infrastructure components. These incidents can be managed through the built-in Veeam Incident Dashboard and resolved manually or automatically with built-in Veeam playbooks.
```
Comment (ShirleyDenkberg) [inline Packs/Veeam/README.md:6]:
```suggestion
Using the data received from Veeam Backup & Replication and Veeam ONE REST APIs, the app creates custom incidents related to malware detection and the health state of the backup infrastructure components. These incidents can be managed through the built-in Veeam Incident dashboard and resolved manually or automatically with built-in Veeam playbooks.
```

File: Packs/Veeam/README.md [hunk 4]
```diff
@@ -17,4 +23,37 @@ The pack includes:
...
+[Veeam Helpcenter User Guide](https://helpcenter.veeam.com/docs/security_plugins_xsoar/guide/)
+
+# Screenshots
+
+![The XSOAR Dashboard](doc_files/Veeam_XSOAR_Dashboard.png)
+
+![Veeam - Start Instant VM Recovery Automatically](doc_files/Veeam_XSOAR_Playbooks.png)
+</~XSOAR>
+<~XSIAM>
+This App allows Veeam Data Platform Advanced and Premium users to monitor various security activities in their Veeam backup infrastructure for:
```
Comment (ShirleyDenkberg) [inline Packs/Veeam/README.md:35]:
```suggestion
This app allows Veeam Data Platform Advanced and Premium users to monitor various security activities in their Veeam backup infrastructure for:
```

File: Packs/Veeam/README.md [hunk 5]
```diff
@@ -17,4 +23,37 @@ The pack includes:
...
+
+- Veeam Backup & Replication events
+- Alarms triggered by Veeam ONE
+
+The App gets information from the event forwarding capabilities via syslog servers integrated with Veeam Backup & Replication and Veeam ONE, parses the data and displays it on the Veeam Data Platform Monitoring dashboard. For events and alarms with Medium, High and Critical severity, the app generates alerts and displays them on the Veeam Security Activities dashboard.
```
Comment (ShirleyDenkberg) [inline Packs/Veeam/README.md:40]:
```suggestion
The app gets information from the event forwarding capabilities via syslog servers integrated with Veeam Backup & Replication and Veeam ONE, parses the data and displays it on the Veeam Data Platform Monitoring dashboard. For events and alarms with Medium, High and Critical severity, the app generates alerts and displays them on the Veeam Security Activities dashboard.
```

File: Packs/Veeam/README.md [hunk 6]
```diff
@@ -17,4 +23,37 @@ The pack includes:
...
+
+The content pack includes:
+
+- Built-in dashboards to monitor job statuses and security activities on a daily basis
+- Built-in reports
+- Multiple data source support
```
Comment (ShirleyDenkberg) [inline Packs/Veeam/README.md:46]:
```suggestion
- Built-in dashboards to monitor job statuses and security activities on a daily basis.
- Built-in reports.
- Multiple data source support.
```

File: Packs/Veeam/README.md [hunk 7]
```diff
@@ -17,4 +23,37 @@ The pack includes:
...
+
+![The XSOAR Dashboard](doc_files/Veeam_XSOAR_Dashboard.png)
+
+![Veeam - Start Instant VM Recovery Automatically](doc_files/Veeam_XSOAR_Playbooks.png)
+</~XSOAR>
+<~XSIAM>
+This App allows Veeam Data Platform Advanced and Premium users to monitor various security activities in their Veeam backup infrastructure for:
+
+- Veeam Backup & Replication events
+- Alarms triggered by Veeam ONE
```
Comment (ShirleyDenkberg) [inline Packs/Veeam/README.md:38]:
```suggestion
- Veeam Backup & Replication events.
- Alarms triggered by Veeam ONE.
```

**Comment by ShirleyDenkberg** [comment]
@rshunim @eepstain Doc review completed.

**Comment by eepstain** [comment]
Hi @nikita-konoplin,
The following needs to be updated in your PR:
- All Dashboards should have a **_fromVersion_** object in them (Attached SS example).
- All Reports should have a **_fromVersion_** object in them (Attached SS example).
- In your pack_metadata.json please add another object- defaultDataSource with a value of either "Veeam ONE REST API" OR "VBR REST API". [Please select the one that's more being used of the two] (Attached SS example).

defaultDataSource
reports
dashboards

Once all of the above will be applied, I'll be able to continue with my checks.

Thank you!

**Comment by nikita-konoplin** [comment]
Hi @eepstain,

Required objects added. Please take a look.

Thank you!

**Comment by eepstain** [comment]
Hi @nikita-konoplin,
Can you please describe the process with which you exported the reports and dashboards?
Currently we are getting error for the **_creator_mail_** object and we would like to verify its origin with you and on our end as well.

**Comment by nikita-konoplin** [comment]
> Hi @nikita-konoplin, Can you please describe the process with which you exported the reports and dashboards? Currently we are getting errors for the **_creator_mail_** object and we would like to verify its origin with you and on our end as well.

Hi @eepstain ,
Reports and dashboards were exported by Yair from his tenant during our call.
And c**_creator_mail_** object was this one: "creator_mail": "yshaya@paloaltonetworks.com", I have only replaced it with "creator_mail": "paloaltoappsupport@veeam.com".

I had this validation error on my local checks too, and I was trying to pass it by returning it back to "creator_mail": "yshaya@paloaltonetworks.com", also, I took a look at dashboards from other packs, they do not have this object at all. I removed these objects, but it still does not pass.

**Comment by nikita-konoplin** [comment]
Hi @eepstain ,

I removed all "creator_mail" objects and now I have new validation errors:

Validating Packs/Veeam/ReleaseNotes/1_0_1.md as releasenotes
Packs/Veeam/ReleaseNotes/1_0_1.md: [RN113] - The content type header "XSIAM Reports" is either an invalid content type or does not exist in the "Veeam" pack.
Please use "demisto-sdk update-release-notes -i Packs/Veeam"
For more information, refer to the following documentation: https://xsoar.pan.dev/docs/documentation/release-notes



Packs/Veeam/ReleaseNotes/1_0_1.md: [RN114] - The xsiamreport "All Veeam malware detection events for the last 24h" does not exist in the "Veeam" pack.
Please use "demisto-sdk update-release-notes -i Packs/Veeam"
For more information, refer to the following documentation: https://xsoar.pan.dev/docs/documentation/release-notes

demisto-sdk update-release-notes -i Packs/Veeam - does nothing
I don't know what the header for XSIAM Reports should look like, could you, please, tell me the right option? 

With [RN114] I also do not know what to do.

Thank you!

**Comment by nikita-konoplin** [comment]
Hi @eepstain 
I found that in other packages, such as here (XSIAMCompliance_HIPAA), these validation errors were ignored, so I did the same.
Please see if I was right.

Thank you!

**Review by eepstain — APPROVED** [review]
Verifying on the vendor's tenant and importing the dashboards to our own tenant.
All of the data seems to be in order. Approved.

**Review by thefrieddan1 — APPROVED** [review]

**Pull request merged**